tests/apollo/test_skvbc_view_change.py

```diff
@@ -533,6 +533,7 @@ async def _wait_for_replica_to_ask_to_leave_view_due_to_status(self, bft_network
             while True:
                 ask_to_leave_msg_count = await bft_network.get_metric(
                     node, bft_network, "Counters", "sentReplicaAsksToLeaveViewMsgDueToStatus")
+                ask_to_leave_msg_count = 0
```

Comment:
I suppose the goal of this small change is to trigger a failure?

Reply:
@teoparvanov , yes, I have mentioned it in the commit message "Commit to check failed logs"

Reply:
Thanks for clarifying!